src/index.js

```diff
@@ -133,14 +133,10 @@ class Kubozer {
       });
     }
 
-    if (optionJS.files === undefined && optionCSS.files === undefined) {
-      throw new Error('WARNING REPLACE(): replace method called but "files" not found in configuration');
-    }
-
     return new Promise((resolve, reject) => {
       try {
-        const changedCSS = replaceInFile.sync(optionCSS);
-        const changedJS  = replaceInFile.sync(optionJS);
+        const changedCSS = replaceInFile.sync(optionCSS.files ? optionCSS : defaultOptions);
```

Comment:
@LasaleFamine maybe you can use Ramda's [merge](http://ramdajs.com/docs/#merge) method